examples/siro_sandbox/app_states/app_state_fetch.py

```diff
@@ -519,6 +519,12 @@ def _get_target_object_position(self, target_obj_idx):
         object_id = self._target_obj_ids[target_obj_idx]
         return rom.get_object_by_id(object_id).translation
 
+    def _get_target_object_bounding_box(self, target_obj_idx) -> mn.Range3D:
```

Comment:
I suppose this and `_get_target_object_position` could both use `_get_target_rigid_object` to avoid code duplication.